data/human/jobs.txt

```diff
@@ -2301,7 +2301,7 @@ mission "Stranded Field Trip"
 	name "Stranded field trip to <planet>"
 	job
 	repeat
-	description `These <bunks> school students have been stranded on a field trip due to the unexpected bankruptcy of their booked transport. The school and parents have put up a reward of <payment> for them to be returned home to <destination>.`
+	description `These <bunks> school students have been stranded on a field trip due to the unexpected bankruptcy of their booked transport. The school and parents have put up a reward of <payment> for them and their Luggage to be returned home to <destination>.`
```

Comment:
```suggestion
	description `These <bunks> school students have been stranded on a field trip due to the unexpected bankruptcy of their booked transport. The school and parents have put up a reward of <payment> for them and <tons> of their luggage to be returned home to <destination>.`
```
The original was already running close to the edge, so you've got to check, yeah. Also, a good name for this PR would be `fix(mission): Update description of human job to mention cargo`

Reply:
![image](https://user-images.githubusercontent.com/20605679/224124392-1f1c7857-7ffb-47bd-9b5b-73d4dd9f184b.png)

Longer planet/system names will be a problem.

We could remove `school`; `school student` is a little tautological.
Could also change `students` to `pupils`.

Instead of:
`The school and parents have put up a reward of <payment> for [...]`
maybe:
`A reward of <payment> is available for [...]`
?

Reply:
```suggestion
	description `These <bunks> pupils have been stranded on a field trip due to the unexpected bankruptcy of their booked transport. A reward of <payment> is available for them and their <tons> of luggage to be returned home to <destination>.`
```